makelib/k8s_tools.mk

```diff
@@ -199,7 +199,7 @@ $(CHAINSAW):
 
 # uptest download and install
 $(UPTEST):
-	@$(INFO) installing uptest $(UPTEST)
+	@$(INFO) installing uptest $(UPTEST_VERSION)
```

Comment:
to be consistent with the other tools, do you want to also update the end `@$(OK) installing uptest $(UPTEST)` message (line 206) to also use `UPTEST_VERSION`?

Reply:
Sure - updated!